javascript/javascript2/week3/Game/game.js

```diff
@@ -0,0 +1,61 @@
+const input = document.getElementById("input");
+const startButton = document.getElementById("startbtn");
+const restartButton = document.getElementById("restartbtn");
+const sKey = document.getElementById("keys");
+const lKey = document.getElementById("keyl");
+const getResult = document.getElementById("result")
+const getAlert = document.getElementById("alert")
+let sCount = 0;
+let lCount = 0;
+
+const startGame = () => {
+
+  if (sCount == lCount) {
+    getResult.innerHTML = "Game Draw!!!";
+  }
+  if (sCount > lCount) {
+    getResult.innerHTML = "S is the Winner !!!";
+  }
+  if (lCount > sCount) {
+    getResult.innerHTML = "L is the Winner";
+  }
+};
+startButton.addEventListener("click", gameStart);
+
+function gameStart() {
+  let getTime = input.value;
+  if (getTime == "") {
```

Comment:
Is good practice to use `===` instead `==`

Reply:
👍🏻